config/database.php

```diff
@@ -44,12 +44,12 @@
             'url' => env('DB_URL'),
             'host' => env('DB_HOST', '127.0.0.1'),
             'port' => env('DB_PORT', '3306'),
-            'database' => env('DB_DATABASE', 'laravel'),
+            'database' => env('DB_DATABASE', 'nbs_db'),
```

Comment:
Please don't work on config/database. Please .env file for this. So Please undo all changes here